README.md

```diff
@@ -67,6 +67,7 @@ Table of Contents
   * [NotABug](https://notabug.org) — NotABug.org is a free-software code collaboration platform for freely licensed projects, Git-based
   * [perforce.com](https://www.perforce.com/products/helix-teamhub) — Free 1GB Cloud and  Git, Mercurial, or SVN repositories.
   * [projectlocker.com](https://projectlocker.com) — One free private project (Git and Subversion) with 50 MB space
+  * [dashboard.ionicframework.com](https://dashboard.ionicframework.com/) - Free repo focused on Ionic Applications
```

Comment:
we'll need a better description of what this actually is, what you get and a link to page that tells us something other than a login form :)

Reply:
I edited it, I had bad URL and I think this description is more clear